pytorch_to_returnn/import_wrapper/context.py

```diff
@@ -118,12 +121,28 @@ def _raise_not_implemented(*args):
     torch.nn.Module: ExplicitWrappedType(torch.nn.Module, WrappedModuleBase, wrap=_raise_not_implemented),
   }
 
+  class Functional(WrappedModuleBase):
+    def __init__(self, func, func_name):
+      super(Functional, self).__init__()
+      self.func = func
+      self.func_name = func_name
+
+    def forward(self, *args, **kwargs):
+      return self.func(*args, **kwargs)
+
+    @classmethod
+    def has_torch_forward(cls) -> bool:
+      return False
+
+  obj_map = {torch.randint: (Functional, (torch.randint, "RandInt"))}
```

Comment:
This is too specific. It should be just a single function, maybe with signature `(obj, *, ctx: WrapCtx, name: str)`.

And the logic you have in `wrap`, you would move here.

Reply:
Like:
```
def _make_wrapped_func(obj, ctx: WrapCtx, name: str):
  wrapped_func = make_wrapped_function(obj, name=name, ctx=ctx)
  wrapped_class = make_wrapped_class(Functional, name="_.Functional", ctx=ctx)
  return wrapped_class(func=wrapped_func, func_name=name)
```

Reply:
Sounds good. Just the name has to be resolved somehow. This way, we cannot `get_module_by_abs_id_name` based on the name `RandInt` from the corresponding module. This has to be hard-coded here somehow, right?

Reply:
What is `name` (`func_name`) actually here? Sth like `torch.rand_int`, right?

Reply:
I used `"RandInd"` because that's the name of the module which does the RETURNN wrapping and therefore when this module looks for the corresponding entry in the namespace it will be by the name `"RandInt"`.

Reply:
But what is `name` here?

Reply:
It's whatever `name` is in `wrap()`. In this specific case, that is `"pytorch_to_returnn.import_wrapper._torch_traced.torch.randint"`.

Reply:
This can be solved by overwriting `RandInt.get_returnn_name` as you suggested below.